tests/benchmark/compute/instruction/test_system.py

```diff
@@ -643,9 +643,6 @@ def test_selfdestruct_created(
     """
     Benchmark SELFDESTRUCT instruction for deployed contracts within same tx.
     """
-    fee_recipient = pre.fund_eoa(amount=1)
```

Comment:
I removed this `fee_receipient` since I can't make sense why this is relevant.
After removing, I didn't get any unexpected error or similar.

So, unless we can figure out why this was here, I would prefer to remove it to avoid confusing future readers.

Reply:
This inits COINBASE as an existing account and thus prevents paying the gas for creating a new account on the first SELFDESTRUCT (25k)

This is an optimization and this cost is amortized over large gas limits.